dataland-frontend/src/components/resources/frameworkDataSearch/DataModelsTranslations.ts

```diff
@@ -0,0 +1,374 @@
+export const lksgKpis = {
+  dataDate: "Data Date",
+  companyLegalForm: "Company Legal Form",
+  vatIdentificationNumber: "VAT Identification Number",
+  numberOfEmployees: "Number Of Employees",
```

Comment:
Auf LkSG Datenseite: Total Revenue soll laut tooltip in Millionen angezeigt werden (aktuell nur eine Zahl), Trenner für 1000er wäre auch gut (also sowas wie 1,534,587.54 statt 1534587.54)

Reply:
Am Ende der Zahl sollte dann auch MM stehen, oder? Sonst erkennt man das ja auch nur über den Tooltip.

Reply:
Ja genau. Das obere Beispiel sollte dann so aussehen: "1,534.59 MM"

Reply:
Ist hinzugefügt.

Reply:
Aber die Nachkommastellen werden noch nicht gerundet. Aktuell würde er "1,534.58754 MM" was unschön aussieht.

Reply:
Ew, Datenverlust. Aber ist gepusht.